src/sql/workbench/contrib/objectExplorer/browser/serverTreeView.ts

```diff
@@ -150,8 +150,8 @@ export class ServerTreeView extends Disposable implements IServerTreeView {
 		return this._actionProvider;
 	}
 
-	public get tree(): ITree | AsyncServerTree {
-		return this._tree!;
+	public get tree(): ITree | AsyncServerTree | undefined {
```

Comment:
Would suggest adding something about why this might be undefined

nit: just removing the ! should be enough - since _tree is already possibly undefined. Although I would update that to be 

`private _tree: ITree | AsyncServerTree | undefined;`

as well since optional doesn't really make much sense there (but functionally it's the same thing)